### DISCUSSION TITLE
New pum broke supertab - how to get autocomplete back?

### DISCUSSION THREAD

**Discussion by zanlib0** [neoclide/coc.nvim#4241] [Q&A] [answered]
It seems like the 0.0.82 update broke supertab integration. I'm not particularly surprised because it's an old plugin, but I'm wondering how to get its way of autocompletion back: tab to go through the completion list, immediately inserting the highlighted option, without the need to confirm, and going to the next option when pressing tab again. I want to be able to use autocomplete only with Tab and S-Tab.

Supertab used to do this seamlessly, but since the introduction of the new popup in 0.0.82 it completely ignores CoC it (even though it seems to input `<C-n>` correctly) and uses the default vim completion window. Disabling vim complete just results in Supertab inserting `^N` into the insert mode.

**Comment by chemzqm** [comment, marked as answer]
Remap `<tab>` by yourself instead.
Checkout the example for copilot https://github.com/neoclide/coc.nvim/wiki/F.A.Q#tab-not-working-well-with-copilotvim
